.github/actions/setup/action.yml

```
@@ -19,4 +19,4 @@ runs:
 
     - name: Install dependencies
       shell: bash
-      run: poetry install
+      run: poetry install --extras pdf
```

Probably safer to just install all extras so this will never need to change in future.
```suggestion
      run: poetry install --all-extras
```